### ISSUE TITLE
why onionscandb/crawls is so larger

### ISSUE THREAD

**Issue by mylamour** [s-rah/onionscan#128]
when i use onionscan to discovery the darknet , i found there was a big problem when the program was running. it's exsume a lot of diskapce
![image](https://cloud.githubusercontent.com/assets/12653147/24742360/8a9fbab8-1adc-11e7-8710-4c31e789454f.png)
you can easily see it on the picture, and i dont why it's so larger. Thanks

**Comment by mylamour** [comment]
After about two weeks, it become to about **100G**

**Comment by s-rah** [comment]
Yes, The snapshot database stores a copy of every page that you visit - this is done to power the correlation engine and can be useful for post-processing of the OnionScan database. There is no way to turn off this behavior without modifying the code.
